packages/app/src/specs/generators/component/ReactComponentGeneratorStepOne.vue

```diff
@@ -0,0 +1,181 @@
+          :loading="query.fetching.value"
+          @select-item="makeSpec"
+        />
+        <GeneratorSuccess
+          v-else
+          :file="result.file"
+        />
+      </div>
+      <div>
+        <StandardModalFooter
+          v-if="result"
+          class="flex gap-16px items-center"
+        >
+          <Button
+            size="lg"
+            :to="{ name: 'SpecRunner', query: { file: posixify(result.file.relative) }, params: { shouldShowTroubleRenderingAlert: true } }"
+            :prefix-icon="TestResultsIcon"
+            prefix-icon-class="w-16px h-16px icon-dark-white"
+            @click="emits('close')"
+          >
+            {{ t('createSpec.successPage.runSpecButton') }}
+          </Button>
+          <Button
+            size="lg"
+            :prefix-icon="PlusButtonIcon"
+            prefix-icon-class="w-16px h-16px icon-dark-gray-500"
+            variant="outline"
+            @click="emits('restart')"
+          >
+            {{ t('createSpec.successPage.createAnotherSpecButton') }}
+          </Button>
+        </StandardModalFooter>
+        <div
+          v-else
+          class="bg-white rounded-b h-24px bottom-0 left-0 w-[calc(100%-24px)] absolute"
+        />
+      </div>
+    </template>
+  </div>
+</template>
+<script setup lang="ts">
+import { useVModels, whenever } from '@vueuse/core'
+import { useI18n } from '@cy/i18n'
+import ExpandableFileChooser from '../ExpandableFileChooser.vue'
+import GeneratorSuccess from '../GeneratorSuccess.vue'
+import { computed, ref } from 'vue'
+import { gql, useQuery, useMutation } from '@urql/vue'
+import type { ComponentGeneratorStepOne_CodeGenGlobFragment, GeneratorSuccessFileFragment } from '../../../generated/graphql'
+import { ReactComponentGeneratorStepOneDocument, ReactComponentGeneratorStepOne_GenerateSpecDocument } from '../../../generated/graphql'
+import StandardModalFooter from '@cy/components/StandardModalFooter.vue'
+import Button from '@cy/components/Button.vue'
+import PlusButtonIcon from '~icons/cy/add-large_x16.svg'
+import TestResultsIcon from '~icons/cy/test-results_x24.svg'
+import EmptyGenerator from '../EmptyGenerator.vue'
+import { posixify } from '../../../paths'
+
+const props = defineProps<{
+  title: string
+  gql: ComponentGeneratorStepOne_CodeGenGlobFragment
+}>()
+const { t } = useI18n()
+const emits = defineEmits<{
+  (event: 'update:title', value: string): void
+  (event: 'update:description', value: string): void
+  (event: 'restart'): void
+  (event: 'close'): void
+}>()
+const { title } = useVModels(props, emits)
+
+title.value = t('createSpec.component.importFromComponent.chooseAComponentHeader')
+
+gql`
+query ReactComponentGeneratorStepOne($glob: String!) {
+  currentProject {
+    id
+    codeGenCandidates(glob: $glob) {
+      id
+      fileName
+      fileExtension
+      absolute
+      relative
+      baseName
+    }
+    # Add the specs, so we can keep the list up to date with the cache
```

Comment:
Do you know if we need this anymore now that we have subscriptions?

Reply:
I'm not sure... this was in the original implementation and I'm not sure what it does